integration_tests/mockApis/prison.ts

```diff
@@ -0,0 +1,62 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from '../wiremock'
+
```

Comment:
As there are two APIs we'll use in prisons - maybe a name change to `prisonerSearchApi` for this one? Other one can be something like `prisonApi`? Just clarifying which is which?